src/components/Sidebar.jsx

```diff
@@ -30,14 +30,14 @@ const SidebarContainer = styled.div`
 
 const chooseLogo = hackathon => {
   switch (hackathon) {
-    case 'hackCamp':
+    case 'hackcamp':
       return hc_logo
-    case 'cmdf':
+    case 'cmd-f':
       return cmdf_logo
-    case 'nwHacks':
-      return nwplus_logo
-    default:
+    case 'nwhacks':
       return logo
```

Comment:
Why don't we use the nwhacks_logo.svg? or if we want to use logo.svg because I see it is a brighter version would it be better to change the filename to nwhacks_logo.svg for consistency? Not sure if hackathon logos change every year- if it does, ignore everything I just wrote LOL

Reply:
turns out the nwHacks logo in logo.svg is super old LOL i just changed it to use the updated one, we can remove the outdated logos in our refactor